<?xml version="1.0"?>
<launch>
    
  <rosparam file="$(find ros_control_test)/config/controllers.yaml" command="load"/>
  <rosparam file="$(find ros_control_test)/config/joint_limits.yaml" command="load"/>
   
  <node pkg="rosserial_python" type="serial_node.py" name="rosserial" output="screen">
    <param name="port" value="/dev/ttyUSB0" />
    <param name="baud" value="2000000" />
  </node>

  <node name="robot_hardware_interface" pkg="ros_control_test" type="ros_control_test_node" output="screen"/>
  
  <!-- node name="subscriber_py" pkg="ros_control_test" type="joints_receive_from_arduino.py" output="screen"/ --> 

  <!-- node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher"/ -->
  
  <!-- node name="rviz" pkg="rviz" type="rviz"/ -->
    
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"
        args="
        	/single_joint_actuator/joints_update
          /single_joint_actuator/joint1_position_controller
        "/>
</launch>
